src/main/java/org/pircbotx/hooks/events/UnknownEvent.java

```diff
@@ -89,4 +89,8 @@ public UnknownEvent(PircBotX bot, String target, String nick, String command, @N
 	public void respond(String response) {
 		getBot().sendRaw().rawLine(response);
 	}
+	
+	public void respondWith(String fullLine) {
+        	getBot().send().message(target, fullLine);
```

Comment:
```suggestion
        	getBot().sendIRC().message(target, fullLine);
```
see[#354](https://github.com/pircbotx/pircbotx/issues/354) for possible deprication